libs/platform/src/lib/components/table/directives/table-cell-resizable.directive.ts

```diff
@@ -0,0 +1,102 @@
+import { AfterViewInit, Directive, ElementRef, HostListener, Input, Optional } from '@angular/core';
+
+import { RtlService } from '@fundamental-ngx/core/utils';
+
+import { TableColumnResizeService } from '../table-column-resize.service';
+
+export type TableColumnResizableSide = 'end' | 'both';
+
+/**
+ * Tracks mouse movement over the cell if the mouse pointer near the side of the cell, informs resize service.
+ */
+@Directive({ selector: '[fdpTableCellResizable]' })
+export class PlatformTableCellResizableDirective implements AfterViewInit {
+    /** First column can be resized only by its end */
+    @Input('fdpTableCellResizable')
+    set resizableSide(value: TableColumnResizableSide) {
+        if (!value) {
+            return;
+        }
+
+        this._resizableSide = value;
+    }
+
+    /** To calculate resized column index */
+    @Input()
+    columnIndex: number;
+
+    /** @hidden */
+    private _resizableSide: TableColumnResizableSide = 'both';
+
+    /** @hidden */
+    constructor(
+        private readonly _elRef: ElementRef,
+        private readonly _rtlService: RtlService,
```

Comment:
Should it be optional?
```suggestion
        @Optional() private readonly _rtlService: RtlService,
```